Pipfile

```diff
@@ -12,3 +12,5 @@ codecov = "*"
 [packages]
 pika = "*"
 structlog = "*"
+pgpy = "*"
+paramiko = "==2.5.0"
```

Comment:
Any reason this is pinned?

Reply:
We were having problems and changed it to see if it would fix it. Can be changed back now